pkg/acsengine/engine_test.go

```diff
@@ -371,3 +371,184 @@ func TestGetGPUDriversInstallScript(t *testing.T) {
 		t.Fatalf("VMSize without GPU should not receive a script, expected empty string, received %v", s)
 	}
 }
+func TestIsNSeriesSKU(t *testing.T) {
+	// VMSize with GPU
+	validSkus := []string{
```

Comment:
Assuming you got this list from somewhere, I have no idea which are which so I'm gonna trust you on this one